setup.cfg

```diff
@@ -20,18 +20,18 @@ classifiers =
 packages = find:
 install_requires =
     PyCifRW~=4.4
-    aiida-core>=2.1,<3
+    aiida-core>=2.2,<3
     aiidalab>=21.11.2
     ansi2html~=1.6
     ase~=3.18
-    bokeh~=2.0
+    bokeh~=2.4
     humanfriendly~=10.0
     ipytree~=0.2
     traitlets~=5.4
     ipywidgets~=7.7
     widgetsnbextension<3.6.3
     pymysql~=0.9
-    nglview~=3.0
+    nglview~=3.0,>=3.0.8
```

Comment:
Earlier patch versions failed to build on python 3.9: https://github.com/aiidalab/aiidalab-widgets-base/actions/runs/19681981549/job/56377980039